### DISCUSSION TITLE
🙋salderen over 3 fases

### DISCUSSION THREAD

**Discussion by spiralshapeturtle** [dsmrreader/dsmr-reader#1743] [Questions & answers / How-to's] [answered]
### Description

Ik heb hier ooit eerder naar gekeken, omdat ik van mening was/ben dat vroeger het verbruik naar 0 ging als je zonne-energie opwekte.

Wat je nu ziet is dat je nog steeds verbruik hebt, ook als je meer opwerkt. Hoe kan ik nu valideren dat mijn meter saldeert over de fases? Stel dat ik 1kwh zonnenergie opwek, en 3x 250kwh afneem op 3 fases. Dan zou in mijn beleving het verbruik 0 moeten zijn (rood) en er 250kwh (groen) teruggeleverd moeten worden aan het net.

Wie weet hoe je dit goed kunt valideren? Hieronder twee screenshots één uit DSMR en één uit homeassistent waar de ververs snelheid 30 seconden is voor de SolarEdge modbus-tcp.

<img width="1219" alt="Greenshot 2022-11-10 12 47 59" src="https://user-images.githubusercontent.com/16855935/201083071-819ab1fd-c6bd-4ea0-a915-7d6451b86014.png">

<img width="404" alt="Greenshot 2022-11-10 12 47 50" src="https://user-images.githubusercontent.com/16855935/201083084-5a3e7e85-f7c5-47bf-8b59-b425027400ad.png">

Als ik hier de som maak:

458 zonne energie
255 verbruik in huis (klopt dat is in de nacht ook ivm servers etc)
405 teruglevering daaruit trek ik de conclusie dat hij niet saldeert over de fases/intern in de meter?

### DSMR-reader version

latest

### DSMR-reader platform

Native (e.g. manual installation)

**Comment by spiralshapeturtle** [comment]
Heb ook Stedin even gecontacteerd, voor nu waar salderen nog 100% is kost het geen geld. Als salderen gaat afbouwen dan is het intern salderen over de meter fases ineens een bron van inkomsten.

**Comment by dennissiemensma** [comment]
Je kunt het beste een telegram uit je meter pakken op het tijdstip. Het zou overeen moeten komen met DSMR-reader, maar dichter bij de bron kan dan bijna niet.
In DSMR-reader kun je het laatste telegram telkens zien in de URL: `/admin/dsmr_datalogger/meterstatistics/`

----

Ik heb zelf geen zonnepanelen, ben geen elektricien en ken de interne hardware van meters niet. Echter krijg ik wel mee dat je niet de enige bent. Vaak ligt dit aan 1 fase teruglevering vs 3 fase verbruik. Ik weet niet of dat ook jouw situatie is.

De meter saldeert volgens mij niet tussen fases en wellicht kan dat technisch ook niet, tenzij je teruglevert over 3 fases (ik ben dus geen elektricien). Als dat het geval is dan is het vooral zaak om zoveel mogelijk stroom zelf te verbruiken om in de toekomst je teruglevering aan het net te drukken. Alleen weet ik dus niet hoeveel je dat gaat besparen en of het uberhaupt de moeite waard is om ermee bezig te zijn.

In België gaan ze overigens al zelfs zo ver dat de netbeheerder extra kosten rekent voor mensen die hoog piekverbruik hebben. Gebruikers met zonnepanelen hebben daarmee de taak om die piek zoveel mogelijk te drukken, waardoor het net ontlast wordt en ze dus ook minder hoeven te betalen aan de netbeheerder.

**Comment by spiralshapeturtle** [comment]
Stedin verwijst naar de energieleverancier, daar komt niet veel zinnigs uit.

Heb nu de PV op fase 2 in het huis geplaatst, dat is de fase met de meeste afnemers waardoor ik de grootste kans heb om de energie zelf af te nemen.

Salderen binnen de meter is altijd een vaag onderwerp, blijft het ook met de afstandelijke houding van Stedin. Het zou fijn zijn als zonnepaneel bezitter als je in DSMR reader een veld kon activeren waar je het totale verbruik van het huis ziet, een positief getal op een donkere dag, en een negatief getal op een zonnige dag.

```
/KFM5KAIFA-METER

1-3:0.2.8(42)
0-0:1.0.0(221115130728W)
1-0:1.8.1(017838.069*kWh)
1-0:1.8.2(012346.293*kWh)
1-0:2.8.1(002762.256*kWh)
1-0:2.8.2(006520.639*kWh)
0-0:96.14.0(0002)
1-0:1.7.0(00.079*kW)
1-0:2.7.0(00.453*kW)
0-0:96.7.21(00005)
0-0:96.7.9(00006)
1-0:99.97.0(7)(0-0:96.7.19)(220302104408W)(0000002192*s)(220218085853W)(0000001912*s)(210404115344S)(0000004978*s)(191122084555W)(0000002077*s)(181215181128W)(0000000208*s)(181026100610S)(0000002253*s)(000101000144W)(2147483647*s)
1-0:32.32.0(00000)
1-0:52.32.0(00001)
1-0:72.32.0(00001)
1-0:32.36.0(00001)
1-0:52.36.0(00001)
1-0:72.36.0(00001)
0-0:96.13.1()
0-0:96.13.0()
1-0:31.7.0(000*A)
1-0:51.7.0(002*A)
1-0:71.7.0(000*A)
1-0:21.7.0(00.008*kW)
1-0:41.7.0(00.000*kW)
1-0:61.7.0(00.072*kW)
1-0:22.7.0(00.000*kW)
1-0:42.7.0(00.450*kW)
1-0:62.7.0(00.000*kW)
0-1:24.1.0(003)
0-1:96.1.0(4730303139333430323736353131303136)
0-1:24.2.1(221115130000W)(04879.456*m3)
!24C4
```

![image](https://user-images.githubusercontent.com/16855935/201916661-39a38efc-7471-4cb7-8aaa-57ed14c54c81.png)

![image](https://user-images.githubusercontent.com/16855935/201916714-19a05a67-8616-4e8d-9141-a4dc829fa775.png)

**Reply by dennissiemensma** [reply]
DSMR-reader ziet alleen wat je meter teruggeeft. Als je je zonnepanelen erbij wilt, kun je kijken naar een oplossing zoals PVOutput (#9). Zowel je omvormer als DSMR-reader pushen dan data daar naar toe.

**Comment by R0yk3** [comment, marked as answer]
Reactie op oude post.
Ik heb hetzelfde gemeld bij Enexis.
Blijkbaar rekent de meter nog na de output van de telegrammen. Het salderen wordt dus pas later berekent in de meter.
Ik heb dit ook eens nagevraagd op tweakers en de versie van Enexis werd daar bevestigd.
